### PR TITLE
Internal: update theme colors

### PR DESCRIPTION
**Pull request by fespinoza**
We needed to sync colors for the different themes.

Also, updated the secondary colors in dark mode to their new values.

### CODE REVIEW

File: Sources/SATSCore/Assets/ThemeColors.xcassets/Gold Theme/Background Top/goldGradientEnd.colorset/Contents.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0.337",
-          "green" : "0.302",
-          "red" : "0.271"
+          "blue" : "0x24",
```
Comment (SatseAndrew) [inline Sources/SATSCore/Assets/ThemeColors.xcassets/Gold Theme/Background Top/goldGradientEnd.colorset/Contents.json:8]: Is this color correct? When I lookup Gradient Gold End (light mode) in the design system i get hex color: 14233C

File: Sources/SATSCore/Assets/ThemeColors.xcassets/Platinum Theme/Primary/platinumPrimaryHighlight.colorset/Contents.json [hunk 2]
```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0.455",
-          "green" : "0.416",
-          "red" : "0.376"
+          "blue" : "0x3C",
```
Comment (SatseAndrew) [inline Sources/SATSCore/Assets/ThemeColors.xcassets/Platinum Theme/Primary/platinumPrimaryHighlight.colorset/Contents.json:8]: Is this color correct? In the design system I find hex color 161718 for Primary Platinum Hover.

File: Sources/SATSCore/Assets/ThemeColors.xcassets/Silver Theme/Background Top/silverGradientStart.colorset/Contents.json [hunk 3]
```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0.455",
-          "green" : "0.416",
-          "red" : "0.376"
+          "blue" : "0x74",
```
Comment (SatseAndrew) [inline Sources/SATSCore/Assets/ThemeColors.xcassets/Silver Theme/Background Top/silverGradientStart.colorset/Contents.json:8]: Is this color correct? I find 14233C for Gradient Silver Start (light mode).

**Review by SatseAndrew — COMMENTED** [review]
I checked the updated colors against the colors Figma and I found some that don't match. Please double check those to see if they are correct or not.

File: Sources/SATSCore/Assets/ThemeColors.xcassets/Gold Theme/Background Top/goldGradientEnd.colorset/Contents.json (hunk 1 above)
Comment (fespinoza) [reply Sources/SATSCore/Assets/ThemeColors.xcassets/Gold Theme/Background Top/goldGradientEnd.colorset/Contents.json:8]: The right color is #262624. Some color names are different that the color values shown

File: Sources/SATSCore/Assets/ThemeColors.xcassets/Silver Theme/Background Top/silverGradientStart.colorset/Contents.json (hunk 3 above)
Comment (fespinoza) [reply Sources/SATSCore/Assets/ThemeColors.xcassets/Silver Theme/Background Top/silverGradientStart.colorset/Contents.json:8]: same here

File: Sources/SATSCore/Assets/ThemeColors.xcassets/Platinum Theme/Primary/platinumPrimaryHighlight.colorset/Contents.json (hunk 2 above)
Comment (fespinoza) [reply Sources/SATSCore/Assets/ThemeColors.xcassets/Platinum Theme/Primary/platinumPrimaryHighlight.colorset/Contents.json]: this was updated after as well

File: Sources/SATSCore/Assets/ThemeColors.xcassets/Gold Theme/Background Top/goldGradientEnd.colorset/Contents.json (hunk 1 above)
Comment (SatseAndrew) [reply Sources/SATSCore/Assets/ThemeColors.xcassets/Gold Theme/Background Top/goldGradientEnd.colorset/Contents.json:8]: Great!

File: Sources/SATSCore/Assets/ThemeColors.xcassets/Platinum Theme/Primary/platinumPrimaryHighlight.colorset/Contents.json (hunk 2 above)
Comment (SatseAndrew) [reply Sources/SATSCore/Assets/ThemeColors.xcassets/Platinum Theme/Primary/platinumPrimaryHighlight.colorset/Contents.json]: In figma? I see that the correct hex code for this is now: 313336.

File: Sources/SATSCore/Assets/ThemeColors.xcassets/Silver Theme/Background Top/silverGradientStart.colorset/Contents.json (hunk 3 above)
Comment (SatseAndrew) [reply Sources/SATSCore/Assets/ThemeColors.xcassets/Silver Theme/Background Top/silverGradientStart.colorset/Contents.json:8]: Ok, can confirm this is correct.

File: Sources/SATSCore/Assets/ThemeColors.xcassets/Platinum Theme/Primary/platinumPrimaryHighlight.colorset/Contents.json (hunk 2 above)
Comment (fespinoza) [reply Sources/SATSCore/Assets/ThemeColors.xcassets/Platinum Theme/Primary/platinumPrimaryHighlight.colorset/Contents.json]: that's the value this color has now. I asked Magnus to update it, before it was the same as the normal primary.

Comment (SatseAndrew) [reply Sources/SATSCore/Assets/ThemeColors.xcassets/Platinum Theme/Primary/platinumPrimaryHighlight.colorset/Contents.json]: I see.

**Review by SatseAndrew — APPROVED** [review]
LGTM 👍

**Pull request merged**